core-web/libs/template-builder/src/lib/components/template-builder/models/models.ts

```diff
@@ -4,20 +4,20 @@ export interface DotGridStackOptions extends GridStackOptions {
     children: DotGridStackWidget[];
 }
 
-export interface DotContainers {
+export interface DotContainer {
     identifier: string;
     uuid: string;
 }
```

Comment:
Rename this to `DotTemplateBuilderContainer` because `DotContainer` we already have.